Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -70,6 +71,7 @@ void CLuaAccountDefs::AddClass(lua_State* luaVM)
 
     lua_classfunction(luaVM, "setData", "setAccountData");
     lua_classfunction(luaVM, "setPassword", "setAccountPassword");
+    lua_classfunction(luaVM, "setSerial", "setAccountSerial");
```

Comment:
This won't work as you are using account name instead of element.
To make it work, use element instead of name or delete this line.
```suggestion
```

Reply:
Good point. At first, I had done it with the account, but I ended up removing it to make it more practical 7418e4c